imports/plugins/core/accounts/client/components/adminInviteForm.js

```diff
@@ -53,6 +53,30 @@ class AdminInviteForm extends Component {
     alertArray: this.state.alertArray.filter((alert) => !_.isEqual(alert, oldAlert))
   });
 
+  sendInvitation = (options) => Meteor.call("accounts/inviteShopMember", options, (error, result) => {
+    if (error) {
+      let messageKey;
+      // switching to use of package i18n keys (groupsInvite. namespace)
+      if (error.reason === "Unable to send invitation email.") {
+        messageKey = "admin.groupsInvite.unableToSendInvitationEmail";
+      } else if (error.reason === "cannot directly invite owner") {
+        messageKey = "admin.groupsInvite.inviteOwnerError";
+      } else if (error.reason === "cannot invite to group") {
+        messageKey = "admin.groupsInvite.cannotInvite";
+      } else if (error.reason === "Need to set a username or email") {
+        messageKey = "admin.groupsInvite.NeedToSetUsernameOrEmail";
+      } else {
+        messageKey = "admin.groupsInvite.errorSendingInvite";
+      }
+      ReactionAlerts.add(error.reason, "danger", Object.assign({}, alertOptions, { i18nKey: messageKey }));
```

Comment:
@loan-laux there is an eslint error and code error on this line, `alertOptions` is not defined. FYI @aldeed